torch/__init__.py

```diff
@@ -759,6 +759,8 @@ def compiled_with_cxx11_abi():
 quantized_lstm = torch.ops.aten.quantized_lstm
 quantized_gru = torch.ops.aten.quantized_gru
 
+from torch.utils.dlpack import from_dlpack, to_dlpack
```

Comment:
Adding some docs for DLPack in gh-70437. I'd forgotten about this. Looked back at my review comment, and it mentioned just `from_dlpack`. I think adding `to_dlpack` to the main namespace was probably unnecessary, given that it shouldn't be used when all libraries support `__dlpack__`. Anyway, no action needed - mostly a comment to self.